rest_framework_bulk/tests/simple_app/views.py

```diff
@@ -23,3 +23,9 @@ def filter_queryset(self, queryset):
 class SimpleViewSet(SimpleMixin, generics.BulkModelViewSet):
     def filter_queryset(self, queryset):
         return queryset.filter(number__gt=5)
+
+
+class SimpleUUIDAPIView(generics.ListBulkCreateUpdateDestroyAPIView):
```

Comment:
can we make this a `ViewSet`? this way we can register it at https://github.com/marcgibbons/django-rest-framework-bulk/blob/feature/uuid-pk-support/rest_framework_bulk/tests/simple_app/urls.py and hence use integration tests or be able to run server locally for testing.